### PR TITLE
Modified according to the Demo review

### PR DESCRIPTION
**Pull request by vecinomio**
Moved all Jenkins related resources from ALB stack to Jenkins stack;
Modified instructions.

### CODE REVIEW

File: ops/cloudformation/jenkins.yml [hunk 1]
```diff
@@ -182,6 +227,7 @@ Resources:
             tar -C /etc/puppetlabs/ -xvf puppet-${pupver}.tar
             retryCommand 5 10 'puppet apply --test /etc/puppetlabs/code/environments/production/manifests/site.pp'
             sleep 30
+            head /var/lib/jenkins/secrets/initialAdminPassword
```
Comment (IYermakov) [inline ops/cloudformation/jenkins.yml]: I strongly beleive that we shouldn't mention any kind of passwords in log. Person who would in charge of Jenkins configuring, definitely had a posibility to read it.

File: ops/cloudformation/jenkins.yml [hunk 2]
```diff
@@ -150,6 +146,55 @@ Resources:
...
+        HttpCode: '200-299'
+      Name: tgJenkins
+      Port: 8080
+      Protocol: HTTP
+      TargetType: instance
+      VpcId:
+        Fn::ImportValue: !Sub '${VPCStackName}-VPCId'
+      Tags:
+      - Key: 'Name'
+        Value: 'tgJenkins'
```
Comment (IYermakov) [inline ops/cloudformation/jenkins.yml]: Maybe we should template it in tg+{Stackname}. What do you think?

Comment (IYermakov) [reply ops/cloudformation/jenkins.yml]: Actually the same for web stack. We should template tags as much as possible (maybe in another PR).

File: ops/cloudformation/jenkins.yml (hunk 1 above)
Comment (vecinomio) [reply ops/cloudformation/jenkins.yml]: Agree. It's not a good solution. Removed

File: ops/cloudformation/jenkins.yml (hunk 2 above)
Comment (vecinomio) [reply ops/cloudformation/jenkins.yml]: Good idea.
Modified according your advice.

**Pull request merged**